packager/packager.h

```diff
@@ -128,6 +128,11 @@ struct StreamDescriptor {
   std::vector<std::string> dash_accessiblities;
   /// Optional for DASH output. It defines Role elements of the stream.
   std::vector<std::string> dash_roles;
+
+  /// Optional for DASH output. Set to true if only DASH output is needed.
```

Comment:
I'd prefer rephrasing it as:

/// Set to true to indicate that the stream is for dash only.

similar below.